src/transceivercontroller/SimulcastTransceiverController.ts

```diff
@@ -98,11 +98,17 @@ export default class SimulcastTransceiverController extends DefaultTransceiverCo
         direction: 'inactive',
         streams: [this.defaultMediaStream],
       });
+
+      /* istanbul ignore next */
```

Comment:
It's because of the safe navigation that we use here `this.meetingSessionContext?.audioProfile.hasRedundancyEnabled()`. I guess the "else" part of the safe navigation is untested. I couldn't find a way to ignore just the safe navigation so I ignored the whole thing instead.

Reply:
I added a test case and removed the `istanbul ignore`